keyboards/pearlboards/pandora/config.h

```diff
@@ -0,0 +1,81 @@
+/*
+Copyright 2021 Koobaczech
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID 0x6963
+#define PRODUCT_ID 0x6963
+#define DEVICE_VER 0x0002
+#define MANUFACTURER Koobaczech
+#define PRODUCT Pandora
+#define DESCRIPTION Premium 60% with cool ass shiz developed by Koobae
```

Comment:
```suggestion
```

Reply:
Fixed the description with a realistic writeup